SpeechResponder/eddi.json

```diff
@@ -546,7 +546,7 @@
\r\n            {return true}\r\n        |else:\r\n            {return false}\r\n        }\r\n    |else: {return false}\r\n    }\r\n}\r\n\r\n{if state.eddi_context_body_type = 'Station' && station.name && station.name != \"\":\r\n    {set current_station to StationDetails(station.name)}\r\n    {set factions_claims to []}\r\n    {set factions_fines to []}\r\n    {set factions_bounties to []}\r\n    {set station_report to false}\r\n    {set system_claims to 0}\r\n    {set system_fines to 0}\r\n    {set system_bounties to 0}\r\n\r\n    {for record in criminalrecord:\r\n        {for faction in system.factions:\r\n            {if record.faction = faction.name:\r\n\r\n                {if record.claims > 0:\r\n                    {set factions_claims to cat(factions_claims, [record.faction])}\r\n                    {set system_claims to system_claims + record.claims}\r\n                }\r\n\r\n                {set reports to record.finesIncurred}\r\n                {if len(reports) > 0:\r\n                    {for report in reports:\r\n                        {if report.shipId = ship.LocalId:\r\n                            {set system_fines to system_fines + report.amount}\r\n                            {if find(factions_fines, record.faction) < 0:\r\n                                {set factions_fines to cat(factions_fines, [record.faction])}\r\n                            }\r\n                        }\r\n                    }\r\n                }\r\n\r\n                {set reports to record.bountiesIncurred}\r\n                {if len(reports) > 0:\r\n                    {for report in reports:\r\n                        {if report.shipId = ship.LocalId:\r\n                            {set system_bounties to system_bounties + report.amount}\r\n                            {if find(factions_bounties, record.faction) < 0:\r\n                                {set factions_bounties to cat(factions_bounties, [record.faction])}\r\n                            }\r\n                        }\r\n                    }\r\n                }\r\n            }\r\n        }\r\n    }\r\n\r\n    {set claim_count to len(factions_claims)}\r\n    {set fine_count to len(factions_fines)}\r\n    {set bounty_count to len(factions_bounties)}\r\n\r\n    {if system_bounties > 0 && HasService(\"Interstellar Factors Contact\"):\r\n        There are legal facilities at this station to clear your pending\r\n        {if bounty_count = 1: bounty |else: bounties}.\r\n        {set station_report to true}\r\n    }\r\n\r\n    {if system_fines > 0 && !station_report:\r\n        {if HasService(\"Contacts\"):\r\n            You may clear your pending {if fine_count = 1: fine |else: fines} at this station.\r\n            {set station_report to true}\r\n        }\r\n    }\r\n\r\n    {if system_claims > 0 && !station_report:\r\n        {if HasService(\"Contacts\"):\r\n            You may claim your pending {if claim_count = 1: reward |else: rewards} at this station.\r\n        }\r\n    }\r\n}",
```

Comment:
There appears to be a problem with the `Crime check station` script at line 17 due to a missing curly bracket: 
Is: 		`{if lcase(ship.size) = "small": {return true}`
Should be: 	`{if lcase(ship.size) = "small": {return true} }`

Reply:
Lol, that's not the only problem. I noticed that error too. The end quote on "medium" was also missing, causing "yellow closing bracket errors". I fixed them both and somehow my pasting of the new script went awry.

Also found similar issue in `Crime check station` that slipped by me.
Fixed.